docs/tools/cli/service/alloydbomni.md

```diff
@@ -0,0 +1,94 @@
+---
+title: avn service alloydbomni
+---
+
+List of commands for `avn service alloydbomni`
+
+## Manage Google service account credentials
+
+Manage a Google service account private key in Aiven for AlloyDB Omni.
+
+### `avn service alloydbomni google-cloud-private-key set`
+
+Add or update a Google service account private key for your Aiven for AlloyDB Omni service.
+
+```bash
+avn service alloydbomni google-cloud-private-key set \
+  --service SERVICE_NAME                             \
```

Comment:
The command here (and everywhere else below) should be without `--service` (i.e. `avn service alloydbomni google-cloud-private-key set --private-key-file PATH_TO_PRIVATE_KEY_FILE SERVICE_NAME`).

Below examples of all 3 commands with their respective outputs:

```
$ avn service alloydbomni google-cloud-private-key set --private-key-file PATH_TO_PRIVATE_KEY_FILE SERVICE_NAME
CLIENT_EMAIL                                                              PRIVATE_KEY_ID                          
========================================================================  ========================================
test-svc-account@test-project-439708.iam.gserviceaccount.com  f336c5bc97043c5c612aa1630447be4412905b23
$
$ avn service alloydbomni google-cloud-private-key show SERVICE_NAME
CLIENT_EMAIL                                                              PRIVATE_KEY_ID                          
========================================================================  ========================================
test-svc-account@test-project-439708.iam.gserviceaccount.com  f336c5bc97043c5c612aa1630447be4412905b23
$
$ avn service alloydbomni google-cloud-private-key delete SERVICE_NAME
Service account key has been removed
```